VK/VK/News/Model/NewsFeedProfile.swift

```diff
@@ -0,0 +1,19 @@
+// NewsFeedProfile.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// News
```

Comment:
```suggestion
///
```
где-то я уже видел такое описание)